mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/cfg/StandalonePojoMapperSettings.java

```diff
@@ -29,12 +31,38 @@ private StandalonePojoMapperSettings() {
 	 */
 	public static final String SCHEMA_MANAGEMENT_STRATEGY = PREFIX + Radicals.SCHEMA_MANAGEMENT_STRATEGY;
 
+	/**
+	 * The mapping configurer to use.
+	 * <p>
+	 * Expects a single-valued or multi-valued reference to beans of type {@link StandalonePojoMappingConfigurer}.
+	 * <p>
+	 * Defaults to no value.
+	 *
+	 * @see org.hibernate.search.engine.cfg The core documentation of configuration properties,
+	 * which includes a description of the "multi-valued bean reference" properties and accepted values.
+	 */
+	public static final String MAPPING_CONFIGURER = PREFIX + Radicals.MAPPING_CONFIGURER;
+
+	/**
+	 * Enables or disables multi-tenancy.
+	 * <p>
+	 * If multi-tenancy is enabled, every {@link SearchMapping#createSession() session} will need to be assigned a tenant identifier.
+	 * <p>
+	 * Expects a boolean value.
+	 * <p>
+	 * Defaults to {@link Defaults#MULTI_TENANCY}.
+	 */
+	public static final String MULTI_TENANCY = PREFIX + Radicals.MULTI_TENANCY;
+
+
 	public static class Radicals {
 
 		private Radicals() {
 		}
 
 		public static final String SCHEMA_MANAGEMENT_STRATEGY = "schema_management.strategy";
+		public static final String MAPPING_CONFIGURER = "mapping.standalone.configurer";
+		public static final String MULTI_TENANCY = "mapping.standalone.multi_tenancy";
```

Comment:
yeah ... my main concern was that we have this in ORM mapper - e.g. https://github.com/hibernate/hibernate-search/blob/main/mapper/orm/src/main/java/org/hibernate/search/mapper/orm/cfg/HibernateOrmMapperSettings.java#L214 and if we use the same property key it might cause a collision at some point... or are we safe? 😃

Reply:
We're safe so far, and it might be a problem in the (distant) future. But like I said, the way to solve this would be to change `PREFIX`, not to introduce fillers here :)